package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "3.8.0",
+  "version": "3.8.1",
```

Comment:
I'll leave the version in my PR, since you're already bumping it up here.

Reply:
Or, the first one of use can go to .1 and the second one to .2
I will adjust my PR to suit whatever you decide in yours.

Reply:
Ok, i'll bump as well 👍